changelog-2024.07.27-8.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index 60f7594158..c73c958ce8 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -115,6 +115,18 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2024.07.27-8" date="2024-07-27">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>Better address search in the USA based on Topologically Integrated Geographic Encoding and Referencing system (TIGER) data. Map size for the USA has increased from 10Gb to 13Gb (including Wikipedia and elevation data)</li>
```

Comment:
This release has only 3 changes compared to the previous one:
1. OpenStreetMap data as of July 23
2. Fixed multiline RTL texts
3. Fixed indentation for Chinese, Japanese and Korean characters when Noto font is used

Reply:
Done